ci-operator/step-registry/baremetalds/sno/rename/baremetalds-sno-rename-commands.sh

```diff
@@ -0,0 +1,317 @@
+#!/usr/bin/env bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+# Fetch packet basic configuration
+# shellcheck source=/dev/null
+source "${SHARED_DIR}/packet-conf.sh"
+
+cat >"${SHARED_DIR}"/recert-rename-sno.sh << "EOF"
+#!/usr/bin/env bash
+
+export PREVIOUS_CLUSTER_NAME="${PREVIOUS_CLUSTER_NAME:-test-infra-cluster}"
+export PREVIOUS_BASE_DOMAIN="${PREVIOUS_BASE_DOMAIN:-redhat.com}"
+export CLUSTER_NAME="${CLUSTER_NAME:-another-name}"
+export BASE_DOMAIN="${BASE_DOMAIN:-another.domain}"
+export SINGLE_NODE_IP="${SINGLE_NODE_IP:-192.168.127.10}"
+
+# assisted-test-infra sets up 2 network interfaces that compete with each other
+# when setting the NODE_IP and KUBELET_NODEIP. Use KUBELET_NODEIP_HINT to
+# ensure the correct interface is chosen.
+#
+# https://access.redhat.com/articles/6956852
+ssh -o UserKnownHostsFile=/dev/null \
+  -o StrictHostKeyChecking=no \
+  -o LogLevel=ERROR \
+  "core@${SINGLE_NODE_IP}" \
+  "echo 'KUBELET_NODEIP_HINT=192.168.127.1' | sudo tee /etc/default/nodeip-configuration"
```

Comment:
Shouldn't this be 192.168.127.10, and can we use the SINGLE_NODE_IP constant we already have?

Reply:
That would be ideal, but unfortunately I don't think we can use the node's IP, as per the documentation: https://access.redhat.com/articles/6956852

> IMPORTANT: Do not use the IP address of a node as a hint, because runtimecfg treats it as a VIP and ignores it when looking for a node IP address, causing the node using the hint IP address to fail to configure correctly.

Reply:
So let's use 192.168.127.0 to make it clearer

Reply:
Sure, I'll update it :+1:

Reply:
How about [this](https://github.com/openshift/release/pull/44454/files#diff-a925b502a3b0c057a42c58f40252cab6b1bf3a04a09c2ba9304141150aeac406R29), does it seem cleaner?

Reply:
You overestimate my understanding of bash

Reply:
I'll make it clearer. I'll give it another go :)

Reply:
How about this instead?

```shell
export SINGLE_NODE_IP="${SINGLE_NODE_IP:-192.168.127.10}"
export SINGLE_NODE_NETWORK_PREFIX="$(echo ${SINGLE_NODE_IP} | cut -d '.' -f 1,2,3).0"
```